flake.nix

```diff
@@ -5,6 +5,7 @@
         flake-parts.follows = "ihp/flake-parts";
         devenv.follows = "ihp/devenv";
         systems.follows = "ihp/systems";
+        ghc-wasm-meta.url = "gitlab:haskell-wasm/ghc-wasm-meta?host=gitlab.haskell.org";
```

Comment:
It would be preferable if we could use Nixpkgs instead of the `ghc-wasm-meta` flake, but [this might not be possible for a while](https://github.com/NixOS/nixpkgs/pull/225000). The JS backend _does_ have support in Nixpkgs, and [will soon be used there for Miso](https://github.com/NixOS/nixpkgs/pull/380737), but I'm a lot more enthusiastic about Wasm in the long run.